src/v/net/connection.h

```diff
@@ -54,12 +54,10 @@ class connection : public boost::intrusive::list_base_hook<> {
     ss::future<> write(ss::scattered_message<char> msg);
     ss::future<> shutdown();
     void shutdown_input();
-    ss::socket_address local_address() const noexcept {
-        return _fd.local_address();
-    }
+    ss::socket_address local_address() const noexcept { return _local_addr; }
 
     // NOLINTNEXTLINE
-    const ss::socket_address addr;
+    const ss::socket_address addr; // remote addr
```

Comment:
why note call it `remote_addr`? then

Reply:
I didn't add that line, just left a comment for clarification, if we rename that should be a second find/replace everywhere PR

Reply:
sounds good.